volumes/miovision/update_intersections/Readme.md

```diff
@@ -221,33 +252,10 @@ We need to find out all valid movements for the new intersections from the data
         leg
     ```
 
-5. **Manual edits**  
-    If you find you need to manually add movements to the above,
-	download the output of the query into a CSV, manually edit the CSV, then
-	append it to `miovision_api.intersection_movements` by modifying the below python snippet, (or use an SQL INSERT statement):
-
-	```python
-	import pandas as pd
-	import psycopg2
-	from psycopg2.extras import execute_values
-
-	import configparser
-	import pathlib
-
-	# Insert code to read configuration settings.
-	postgres_settings = {your_postgres_config}
+5. **Update geojson**
+	Update the [geojson intersections file](../geojson/mio_intersections.geojson) by exporting to file from QGIS with `GeoJSON` format. This geojson file is helpful as a publically accessible record of our Miovision intersections.   
+![export to file from QGIS](geojson_export.png)
 
-	# Insert the name of your CSV file.
-	df = pd.read_csv({your_file.csv})
-	df_list = [list(row.values) for i, row in df.iterrows()]
-
-	with psycopg2.connect(**postgres_settings) as conn:
-		with conn.cursor() as cur:
-			insert_data = """INSERT INTO miovision_api.intersection_movements(intersection_uid, classification_uid, leg, movement_uid) VALUES %s"""
-			execute_values(cur, insert_data, df_list)
-			if conn.notices != []:
-				print(conn.notices)
-	```
 ## Update `miovision_api.centreline_miovision`
 
 [`miovision_api.centreline_miovision`](../README.md) links Miovision intersection legs to `gis.centreline` street segments. The script used to create this table is [here](../sql/table/create-mv-mio_cent.sql). 
```

Comment:
gis_core

Reply:
I added a note in the centreline_miovision section of 'sql' readme that this table needs to be updated to refer to gis_core.